source/reference/operator/aggregation/geoNear.txt

```diff
@@ -204,8 +204,147 @@ When using :pipeline:`$geoNear`, consider that:
 - Starting in version 4.2, :pipeline:`$geoNear` no longer has a default
   limit of 100 documents.
 
-Example
--------
+- Starting in MongoDB 5.1, the ``near`` parameter supports the 
+  :ref:`let option <geoNear_let_example>` and
+  :ref:`bound let option <geoNear_bounded_let_example>`.
+
+Examples
+--------
+
+.. _geoNear_let_example:
+
+$geoNear with the ``let`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example:
+
+- The ``let`` option is used to set an array value of 
+  ``[-73.99279,40.719296]`` to the variable ``$pt``.
+
+- ``$pt`` is a let option to the ``near`` parameter to the ``$geoNear`` 
+  stage.
+
+.. code-block:: javascript
+   :emphasize-lines: 6,16
+
+   db.places.aggregate(
+   [
+      {
+         "$geoNear":
+         {
+            "near":"$$pt",
+            "distanceField":"distance",
+            "maxDistance":2,
+            "query":{"category":"Parks"},
+            "includeLocs":"dist.location",
+            "spherical":true
+         }
+      }
+   ],
+   {
+      "let":{ "pt": [ -73.99279, 40.719296 ] }
+   }
+   )
+
+The aggregation returns the following:
```

Comment:
Can we explain a little bit what the results mean? The other examples on this page have explanations like:

> The following aggregation uses $geoNear to find documents with a location at most 2 meters from the center [ -73.99279 , 40.719296 ] and category equal to Parks.

Users might have an easier time following these examples if we briefly explain what the goal of the pipeline we've created is.

Reply:
I specified this above each of the queries : 

In this example, :pipeline:`$lookup` uses:

- ``let`` to define ``$pt``.
- :pipeline:`$geoNear` in the ``pipeline``.
- ``$pt`` to define ``near`` in the :pipeline:`$geoNear` pipeline stage.

In this example:

- The ``let`` option is used to set an array value of 
  ``[-73.99279,40.719296]`` to the variable ``$pt``.

- ``$pt`` is specified as a let option to the ``near`` parameter in the 
  ``$geoNear`` stage.
  
  Do you mean explictly changing "the aggregation returns the following" to describe the pipeline? I tried to  copy the existing format as much as possible.

Reply:
Yeah I saw, but my point was more that we've explained what the parameters are but not how they affect the end result. I think if we adjusted:

> The aggregation returns the following:

To be something like:

```
The aggregation returns all documents with: 

- A location at most 2 meters from the point defined in  the ``let`` variable
- A ``category`` equal to ``Parks``.

.. code-block::
```

it could help paint a clearer picture. The examples which were on the page prior to this change do have an explanation like this. However, if you think the current approach is clear enough we can leave this alone.